README.md

```diff
@@ -35,40 +35,38 @@ structure defined in the default stack.
 
 ### Prerequisites
  - Python 3.8+
- - [Cookiecutter Python package](http://cookiecutter.readthedocs.org/en/latest/installation.html) >= 2.1.0: This can be installed with pip:
+ - [Databricks CLI](https://docs.databricks.com/en/dev-tools/cli/databricks-cli.html) >= v0.203.2
```

Comment:
Since you're using the `order` field in the template schema, you need `v0.204.0`.